docker/prod-compose.yml

```diff
@@ -110,41 +62,41 @@ services:
     working_dir: /var/www/html
 
   php-fpm:
-    build: ./php
-    restart: always
-    env_file: ../facebook.env
+    # env_file: ../facebook.env
     environment:
+      FACEBOOK_APP_ID: ${FACEBOOK_APP_ID?fb_app_id}
+      FACEBOOK_APP_SECRET: ${FACEBOOK_APP_SECRET?fb_app_secret}
       MYSQL_HOST: ${MYSQL_HOST?host}
```

Comment:
Out of curiosity what's the point of adding these defaults? Wouldn't it be a disabling issue if one of these env vars are undefined?

Reply:
Yep, this doesn't add a default but instead specifies a label. For instance, running without it specifies gives an error output:
```
Missing mandatory value for "environment" option interpolating {'FACEBOOK_APP_ID': '${FACEBOOK_APP_ID?fb_app_id}', 'FACEBOOK_APP_SECRET': '${FACEBOOK_APP_SECRET?fb_app_secret}'} in service "php-fpm": fb_app_id
```

Reply:
Makes sense, thank you!